types/react-router/v3/react-router-tests.tsx

```diff
@@ -222,16 +222,20 @@ match({ history, routes }, (error, redirectLocation, renderProps) => {
     renderToString(<RouterContext {...renderProps} />);
 });
 
-ReactDOM.render((
+ReactDOM.render(
     <Router
         history={history}
         routes={routes}
-        render={applyRouterMiddleware({
```

Comment:
`applyRouterMiddleware` doesn't return render-props i.e. an element factory but a component factory. This only worked because `ComponentType` was assignable to `{}`